common/ai_budget/00_minerals_budget.txt

```diff
@@ -0,0 +1,643 @@
+				value < 1000
+			}
+			resource_stockpile_compare = {
+				resource = minerals
+				value > 10000
+			}
+			resource_income_compare = {
+				resource = minerals
+				value > 200
+			}
+		}
+	}
+}
+
+minerals_expenditure_stations_2 = {
+	resource = minerals
+	type = expenditure
+	category = stations
+
+	potential = {
+		always = yes
+		resource_stockpile_compare = { resource = minerals value >= 800 }
+		resource_stockpile_compare = { resource = minerals value < 1200 }
+	}
+
+	fraction = {
+		weight = 0.25
+	}
+}
+
+minerals_expenditure_stations_3 = {
+	resource = minerals
+	type = expenditure
+	category = stations
+
+	potential = {
+		always = yes
+		resource_stockpile_compare = { resource = minerals value >= 1200 }
+	}
+
+	fraction = {
+		weight = 0.2
+	}
+}
+
+minerals_expenditure_relic_origin = {
+	resource = minerals
+	type = expenditure
+	category = deposit_blockers
+	potential = {
+		years_passed > 5
+		resource_stockpile_compare = {
+			resource = minerals
+			value > 600
+		}
+		resource_income_compare = {
+			resource = physics_research
+			value > 80
+		}
+	}
+
+	fraction = {
+		weight = 0.7
+	}
+
+	static_min = {
+		base = 0
+		modifier = {
+			add = 900
+			years_passed > 3
+		}
+	}
+}
+
+minerals_expenditure_colonies_lithoid_expand = {
+	resource = minerals
+	type = expenditure
+	category = colonies
+
+	potential = {
+		is_machine_empire = no
+		is_lithoid_empire = yes
+		ai_colonize_plans > 0
+		has_resource = { type = minerals amount > 400 }
+		has_monthly_income = {
+			resource = minerals
+			value > 0
+		}
+	}
+
+	fraction = {
+		weight = 0.5
+	}
+
+	static_min = {
+		base = 550
+	}
+}
+
+minerals_expenditure_armies = {
+	resource = minerals
+	type = expenditure
+	category = armies
+
+	potential = {
+		or = {
+			resource_stockpile_compare = {
+				resource = minerals
+				value > 1500
+			}
+			and = {
+				resource_stockpile_compare = {
+					resource = minerals
+					value > 600
+				}
+				resource_income_compare = {
+					resource = minerals
+					value > 45
+				}
+			}
+			and = {
+				has_origin = origin_necrophage
+				years_passed < 7
+				years_passed > 3
+			}
+		}
+		or = {
+			is_at_war = yes
+			and = {
+				prepare_for_war = yes
+				used_naval_capacity_percent <= 1
+			}
+			and = {
+				resource_stockpile_compare = {
+					resource = minerals
+					value > 4000
+				}
+				resource_income_compare = {
+					resource = minerals
+					value > 50
+				}
+				resource_income_compare = {
+					resource = energy
+					value > 10
+				}
+				used_naval_capacity_percent > 0.9
+			}
+			and = {
+				count_owned_army = {
+					limit = {
+						NOR = {
+							army_type = robotic_defense_army
+							army_type = defense_army
+							army_type = machine_defense
+						}
+					}
+					count < 3
+				}
+				has_origin = origin_necrophage
+			}
+			and = {
+				count_owned_army = {
+					limit = {
+						NOR = {
+							army_type = robotic_defense_army
+							army_type = defense_army
+							army_type = machine_defense
```

Comment:
I remember special defense armies, but that be something that will be implemented in the future.